cockroachdb/kustomization.yaml

```diff
@@ -1,17 +0,0 @@
-apiVersion: kustomize.config.k8s.io/v1beta1
```

Comment:
Pipeline is failing, because there is this file referencing non-existing files on `main` branch

Reply:
> I guess we should have a second look on our pipeline- it was not reported on [original branch](https://github.com/utilitywarehouse/shared-kustomize-bases/pull/39)

Fixing https://github.com/utilitywarehouse/shared-kustomize-bases/pull/45

Reply:
Thank you!